README.md

```diff
@@ -43,9 +43,20 @@ Pixi.js has full [WebGL](https://en.wikipedia.org/wiki/WebGL) support and seamle
 - Chat: You can join us on [Gitter](https://gitter.im/pixijs/pixi.js) To chat about Pixi. We also now have a Slack channel. If you would like to join it please Send me an email (mat@goodboydigital.com) and I will invite you in.
 
 
-### Installation ###
+### Setup ###
 
-Pixi.js can be installed with [Bower](https://bower.io/#getting-started), [NPM](https://docs.npmjs.com/getting-started/what-is-npm) or simply using a content delivery network (CDN) URL to embed Pixi.js directly on your HTML page. 
+Its easy to get started with Pixi.js! Simple grab the pre-built versions from here:
+
+Master - Nice and stable Pixi.js
+- Unminified: [http://pixijs.download/master/pixi.js]
+- Minified: [http://pixijs.download/master/pixi.min.js] 
```

Comment:
I would change this to use the **release** branch instead **http://pixijs.download/release/pixi.js**, this will be the most stable.